EssentialsGeoIP/pom.xml

```diff
@@ -26,14 +26,57 @@
 
     <dependencies>
         <dependency>
-            <groupId>net.ess3</groupId>
```

Comment:
You removed essentials from the dependencies. Breaks this.

Reply:
Some how maven include the whole ess into GeoIP.jar with the dependency. To be honest I'm not sure how to do it properly. At least it doesn't cause problem when I tested it :P